### PR TITLE
add libremap as an external package

### PR DESCRIPTION
**Pull request by aparcar**
this will automatically build libremap via ci

### CODE REVIEW

File: packages/libremap-agent/Makefile [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=libremap-agent
+PKG_VERSION:=0.1.9
+PKG_RELEASE=$(PKG_SOURCE_VERSION)
+
+
+PKG_SOURCE_URL:=git://github.com/aparcar/libremap-agent-openwrt.git
```
Comment (p4u) [inline packages/libremap-agent/Makefile]: Shouldn't it be HTTPs?

**Comment by aparcar** [comment]
no

File: packages/libremap-agent/Makefile (hunk 1 above)
Comment (aparcar) [reply packages/libremap-agent/Makefile]: no

Comment (gmarcos87) [reply packages/libremap-agent/Makefile]: Shouldn't it be https://github.com/libremesh/libremap-agent.git ?

Comment (aparcar) [reply packages/libremap-agent/Makefile]: In order to be able to build it I moved everything to the top folder https://github.com/aparcar/libremap-agent-openwrt/commit/054133cd07b6ca77bba32537ad2141a193a94111

Comment (aparcar) [reply packages/libremap-agent/Makefile]: Once this is upstream we could change it back

**Pull request closed without merging**